environments/common/layouts/everything

```diff
@@ -82,6 +82,10 @@ openhpc
 # Hosts to run ansible-init
 cluster
 
+[compute_init:children]
+# Hosts to deploy compute initialisation ansible-init script to.
```

Comment:
I don't think this description is right.

Deploying the actual compute init script is/will be done in the image build.

This should control which hosts get info templated out (and eventually, metadata set to turn on the feature, I think).